test/drive.js

```diff
@@ -128,6 +149,41 @@ test('Multiple Archives File', function (t) {
   })
 })
 
+test('Multiple Archive POST File', function (t) {
+  var onrequest = hyperdriveHttp(getArchive)
+  server.once('request', onrequest)
+  fs.createReadStream(path.join(__dirname, 'drive.js'))
+  .pipe(request.post('http://localhost:8000/', function (err, res, body) {
```

Comment:
Sweet! I think that means this works instead of my open PR.

We should add this to docs.